uw-frame-components/portal/settings/partials/user-settings.html

```diff
@@ -9,25 +9,15 @@ <h4 class="md-subhead" ng-if="kvEnabled">Reset Zone<br>
         <small>Reset various features to their default states.</small>
       </h4>
       <div ng-if="kvEnabled" layout="column" layout-align="center start">
-        <!-- RESET ANNOUNCEMENTS -->
-        <div layout-xs="column" layout-align-xs="center start" layout="row" layout-align="start center" ng-if="FEATURES.enabled">
+        <!-- RESET MESSAGES -->
+        <div layout-xs="column" layout-align-xs="center start" layout="row" layout-align="start center" ng-if="MESSAGES.announcementsEnabled || MESSAGES.notificationsEnabled">
           <md-button class="md-primary md-raised"
-                     ng-click="resetAnnouncements()"
+                     ng-click="resetMessages()"
                      aria-label="reset seen announcements">
-            Reset announcements
-            <i class="fa fa-spin fa-circle-o" ng-if="loadingResetAnnouncements"></i>
+            Reset in-app messages
```

Comment:
Calling portal-wide messages "in-app messages" is going to confuse with, well, in-app messages ala the within-specific-HRS-app in-app messages we've been doing.